### PR TITLE
Troubleshooting inspector guide review

### PR DESCRIPTION
**Pull request by mattw59**

### CODE REVIEW

File: source/ember-inspector/troubleshooting.md [hunk 1]
```diff
@@ -1,42 +1,37 @@
-Below are some common issues you may encounter, along with the necessary
-steps to solve them. If your issue is not listed below, please submit an
-issue to the inspector's [Github repo][ember-inspector-github].
+Below are some common issues you may encounter in using the Inspector, along with the
```
Comment (locks) [inline source/ember-inspector/troubleshooting.md]: `when using the Inspector` maybe?

File: source/ember-inspector/troubleshooting.md [hunk 2]
```diff
@@ -1,42 +1,37 @@
...
+steps necessary to solve them. If your issue is not listed below, please submit an
```
Comment (locks) [inline source/ember-inspector/troubleshooting.md]: I prefer `necessary steps`

File: source/ember-inspector/troubleshooting.md [hunk 3]
```diff
@@ -60,23 +52,20 @@ This happens if you are using a version of Ember < 1.3.
 
 #### Missing Promises
 
-If the Promises tab is working, but there are promises you can't find,
+If the Promises tab is working, but there are Promises you can't find,
```
Comment (locks) [inline source/ember-inspector/troubleshooting.md]: the guides are starting to feel like german xD

File: source/ember-inspector/troubleshooting.md [hunk 4]
```diff
@@ -60,23 +52,20 @@ This happens if you are using a version of Ember < 1.3.
...
 
 <img src="../../images/guides/ember-inspector/troubleshooting-promises-toolbar.png" width="350">
 
 #### Inspector Version Old on Firefox
 
-Firefox addons need to go through a review process before every update.
-This causes the Firefox Ember Inspector to usually be one version
-behind.
+Firefox addons need to go through a review process with each update.
+The Firefox Ember Inspector is usually one version behind.
```
Comment (locks) [inline source/ember-inspector/troubleshooting.md]: Unnecessary simplification, sentences feel more disjoint.

File: source/ember-inspector/troubleshooting.md (hunk 3 above)
Comment (mattw59) [inline source/ember-inspector/troubleshooting.md]: Danke

**Comment by mattw59** [comment]
Have another look when you get a chance, @locks. Thanks for your feedback!

**Comment by michaelrkn** [comment]
Thanks @matt59!

**Pull request merged**